.github/issue_template.md

```diff
@@ -0,0 +1,5 @@
+### Expected Behavior
```

Comment:
Actually, question! When creating an issue, the form fields are essentially Title + Description. Should we repeat those fields in the template?

<img width="1033" alt="screen shot 2017-05-06 at 1 25 31 pm" src="https://cloud.githubusercontent.com/assets/386833/25775719/9cb98bb6-3260-11e7-85fa-54e2d9fc5acb.png">